### DISCUSSION TITLE
Separate Image Cache

### DISCUSSION THREAD

**Discussion by Undistraction** [gatsbyjs/gatsby#29027] [Ideas / Feature Requests]
I often need to delete the cache during development, either because of compilation bugs, or to as a sanity check to narrow down what might be compilation bugs. When I do this, I'm faced with a full rebuild, which currently takes ~6 mins. About 5 mins of that is taken up with image processing ('Generating image thumbnails'), however images are almost never the problem. 

Would it be feasible to make the cache more granular, so that we could delete e.g. the page cache independently from the image cache?

**Comment by jgosmann** [comment]
I have a website with a lot and large images. A full rebuild even takes me a full hour at this point. It also seems that an upgrade of the Gatsby packages will also invalidate the cache. I'm also desperately looking for something to alleviate this situation.
